client/src/components/Dashboard.js

```diff
@@ -157,16 +179,24 @@ class Dashboard extends Component {
       <Container>
         <Header style={{ marginTop: "25px" }} as="h2">
           Good {getGreeting()} {userName}!
+          {userRole === "User" ? (
+            <Button
+              style={({ marginBottom: "1em" }, { float: "right" })}
+              color="blue"
+              onClick={() => this.props.history.push("/add-document")}
+            >
+              Add document
+            </Button>
+          ) : (
+            <Checkbox
+              style={{ float: "right" }}
+              toggle
+              label="Only show documents I can translate"
+              onClick={() => this.handleToggle(languages)}
+            />
+          )}
```

Comment:
To fix the style for mobile, first extract all this logic in a separate function `getDashboardOptions` (for example) and remove all the `float: right`. Then try something like this:

```
<Header style={{ marginTop: "25px" }} as="h2">
  Good {getGreeting()} {userName}!
  <Responsive {...Responsive.onlyMobile} style={{margin: "1em 0"}}>
      {this.getDasboardOptions()}
  </Responsive>
  <Responsive minWidth={Responsive.onlyTablet.minWidth} style={{ float: "right" }}>
      {this.getDasboardOptions()}
  </Responsive>
</Header>
```

Reply:
done